### DISCUSSION TITLE
An option to disable the whole "This time, search with" bar

### DISCUSSION THREAD

**Discussion by awsms** [zen-browser/desktop#1535] [Ideas]
When you've set up many custom search engines and press Ctrl+L, you will be presented a ridiculous amount of favicons that show all your personalized search engines in the "This time, search with:" submenu.
An option to completely turn it off would be very welcome, the only alternative right now being to disable every single custom search engine, or to hide the panel (but then it can be triggered if pressing Tab).
https://support.mozilla.org/en-US/questions/1315296#answer-1370408

**Comment by davidmickelsen** [comment]
I wholeheartedly +1 this request. A "ridiculous amount of favicons" perfectly describes what I'm dealing with.

**Comment by 12th-devs** [comment]
@awsms if you want to do this right now you can write custom css to achieve this, just highlight over the parent container and use the listed id then add display: none ! important; in user chrome.css

Read Zen documentation to understand user chrome.css
